e2e/fixtures/events/createClaim.js

```diff
@@ -141,9 +141,17 @@ module.exports = {
       PaymentReference: {
         paymentReference: 'Applicant reference'
       }
+    },
+    invalid:{
+      Court: {
+        courtLocation: {
+          applicantPreferredCourt: '3a3'
```

Comment:
should we also check if longer than 3 characters? Can the court code be shorter? aka 21, or would it be 021?

Reply:
following the regex pattern being used ( "[0-9]{3}") it would have to be 021 in that case